src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java

```diff
@@ -91,6 +93,7 @@ public void evaluate() throws Throwable {
                     restartRequired |= installPlugins(d.getTestClass().getAnnotation(WithPlugins.class), plugins);
                     LOGGER.info("for " + d + " asked to install " + plugins + "; restartRequired? " + restartRequired);
                     if (restartRequired) {
+                        assumeTrue("This test requires a restartable Jenkins", jenkins.canRestart());
```

Comment:
would be nice to know this ahead of time - but how do you find this out before Jenkins starts :-)